tests/scripts/quiet/quiet

```diff
@@ -0,0 +1,66 @@
+#! /usr/bin/env bash
+#
+# Copyright The Mbed TLS Contributors
+# SPDX-License-Identifier: Apache-2.0 OR GPL-2.0-or-later
+#
+# This swallows the output of the wrapped tool, unless there is an error.
+# This helps reduce excess logging in the CI.
+
+# If you are debugging a build / CI issue, you can get complete unsilenced logs
+# by un-commenting the following line (or setting VERBOSE_LOGS in your environment):
+# VERBOSE_LOGS=1
+
+# Locate original tool
+TOOL_WITH_PATH=$(dirname "$0")/$TOOL
+ORIGINAL_TOOL=$(type -ap "${TOOL}" | grep -v -Fx "$TOOL_WITH_PATH" | head -n1)
+
+print_quoted_args() {
+    # similar to printf '%q' "$@"
+    # but produce more human-readable results for common/simple cases like "a b"
+    for a in "$@"; do
+        simple_pattern='^([[:alnum:]_+-]+=)?([[:alnum:] _=+-./:@]*)$'
```

Comment:
`+-./` in the character set looks like a gotcha. It's the range from `+` to `.` and the character `/`. The range from `+` to `.` is the four characters `+,-.` — the set includes the comma but you might miss it at first glance. If you're going to use ranges, you might as well use `+-/` since `/` follows `.`. However, all of these observations about ranges are true only if the ASCII punctuation symbols are in order and with no other character interspersed. I don't know if that's the case in practice. It would be safer to avoid ranges. A hyphen does not denote a range if it's immediately after the opening `]` or immediately before the closing `]`.
```suggestion
        simple_pattern='^([[:alnum:]_+-]+=)?([-[:alnum:] _=+./:@]*)$'
```